jujugui/static/gui/src/app/views/utils.js

```diff
@@ -2097,6 +2097,56 @@ YUI.add('juju-view-utils', function(Y) {
     /*jshint debug:false */
   });
 
+  Y.Handlebars.registerHelper('compare', function(lvalue, rvalue, options) {
+    if (arguments.length < 3) {
+      throw new Error('Handlerbars Helper "compare" needs 2 parameters');
+    }
+    var operator = options.hash.operator || '==';
+    var operators = {
+      '==':       function(l,r) { return l == r; },
+      '===':      function(l,r) { return l === r; },
+      '!=':       function(l,r) { return l != r; },
+      '<':        function(l,r) { return l < r; },
+      '>':        function(l,r) { return l > r; },
+      '<=':       function(l,r) { return l <= r; },
+      '>=':       function(l,r) { return l >= r; },
+      'typeof':   function(l,r) { return typeof l == r; }
+    };
+    if (!operators[operator]) {
+      throw new Error('Handlerbars Helper "compare" does not know the operator '+operator);
+    }
+    var result = operators[operator](lvalue,rvalue);
+    if (result) {
+      return options.fn(this);
+    } else {
+      return options.inverse(this);
+    }
+  });
+
+  Y.Handlebars.registerHelper('fullSeries', function(series) {
+    series = series.toLowerCase();
+    if (series === 'oneiric') {
```

Comment:
instead of using all of these conditionals why not use a map?
```JavaScript
var names = {
    oneric: 'Oneric 11.10',
    ...
};
return names[series] || series.charAt(0).toUpperCase() + series.slice(1);
```

although I'd probably use css to uppercase instead of doing it here...